### ISSUE TITLE
When starting PI, create a warning if there are prior PI results

### ISSUE THREAD

**Issue by StephanSchaller** [Open-Systems-Pharmacology/OSPSuite.Core#1401]
If you re-run a PI (by accident) you will loose your prior results (even if you cancel, the results produced until then are overwriting the old results). Also, no rollback exists for the PI, so it would be good to have an extra confirmatory step before starting a PI if prior results exist.

**Comment by msevestre** [comment]
Hum....how do you start a PI by accident? 

We do not ask this question when starting a simulation or a pop simulation. I don't think we should for a PI.

**Comment by PavelBal** [comment]
I personally also do not really like the proposal... However, I see the problem that PI results are sometimes lost (e.g. because you started a new one and then realized the results are worse...). I would propose to have a feature to export the PI results. The easiest way would be probably to export to excel, with first sheet having all the parameters and their values (have to think about how to present linked parameters...), and then (final) simulation results and corresponding observed data in separate sheets.

A more comprehensive solution would be to export PI results to PKML with an option to load them again.

**Comment by msevestre** [comment]
What about cloning a PI? That's....a good use case ™ for this requirement.

**Comment by PavelBal** [comment]
Out of experience - this does not work well. Not in terms that it technically does not work, the workflow is just not good enough. It leads to multiple PI instances and it is easy to loose track of them.

**Comment by StephanSchaller** [comment]
> Hum....how do you start a PI by accident?

well you accidently click the "play" button (maybe you think you are in a simulation and want to run it an then its too late when you realize you are not... no way to undo this, old results lost forever...)... It is just a mini Warning Prompt. It is not like this will increase user time to run a PI as you don't run PIs every second for 1000 times and this extra click costs time. But it may save a lot of time if it keeps you from starting the PI by accident...

**Comment by StephanSchaller** [comment]
> We do not ask this question when starting a simulation or a pop simulation. I don't think we should for a PI.

well, you run single simulations very often. An extra click makes this annoying. 
For popSims I think it could make sense, too.
Maybe this can be an option you can set?

**Comment by msevestre** [comment]
I disagree with all those extra prompt. This is my personal preference from a user experience. I think if the project is soo important, you should have a backup that you could go back to. Or you just close your project and don't save...sorry, nothing you said is convincing
